src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIOTalonFX.java

```diff
@@ -117,23 +143,27 @@ public void updateInputs(ShooterIOInputs inputs) {
     boolean leftConnected =
         BaseStatusSignal.refreshAll(
                 leftPosition,
```

Comment:
The ```StatusSignal``` objects from CTRE are in units of Rotations.


```suggestion
    public StatusSignal<Double> rightVelocityErrorRotationsPerSecond;;
```